open-craft/accounting
# Automate TransferWise Bulk Payment CSV Generation

This PR adds an automated job that generates a TransferWise Bulk Payment CSV every month for all unpaid invoices hitherto, and sends that CSV to a pre-set account.

**Testing Instructions**:

1. If necessary, create an account with username `opencraft` (or whatever you set `TRANSFERWISE_BULK_PAYMENT_SENDER` to), with an email you have access to.
1. If necessary, create 1 or 2 provider accounts, and as usual, all of the scaffolding, like hourly rates, invoice templates, and bank accounts.
1. If necessary, run the tasks in `accounting.invoice.tasks`, so that invoices are generated for the providers, and are populated with line items from JIRA.
1. Go ahead and run the task introduced here, in `accounting.transferwise.tasks`.
1. Ensure that the client receives an email with the CSV attached, and that the CSV contains the correct rows and information. Also check that it is uploaded in the appropriate GDrive folder, where the invoices are.
